src/main/java/brandkon/brand/BrandController.java

```diff
@@ -0,0 +1,28 @@
+package brandkon.brand;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class BrandController {
+
+    private final BrandService brandService;
+
+    public BrandController(BrandService brandService) {
+        this.brandService = brandService;
+    }
+
+    @GetMapping("/brands")
+    public List<BrandResponse> readBrands(@RequestParam(required = false) String category){
```

Comment:
참고로 아래처럼 `@RequestParam(required = false)`가 생략되어도 동작은 같습니다.
```suggestion
    public List<BrandResponse> readBrands(String category){
```